openwpm/deploy_browsers/deploy_firefox.py

```diff
@@ -157,4 +158,4 @@ def deploy_firefox(status_queue, browser_params, manager_params, crash_recovery)
 
     status_queue.put(("STATUS", "Browser Launched", int(pid)))
 
-    return driver, driver.capabilities["moz:profile"]
+    return driver, driver.capabilities["moz:profile"], display
```

Comment:
Could you please give this function a type annotation.

Reply:
I think I annotated this properly--unsure, though